gateware/vga/testcapture.py

```diff
@@ -0,0 +1,199 @@
+from migen.fhdl.std import *
```

Comment:
This is a test bench right? Can you add a python docstring at the top of this file?